scripts/extensions/sams/src/store/assets/actions.ts

```diff
@@ -44,6 +46,19 @@ export function setAssetListStyle(style: ASSET_LIST_STYLE): IAssetActionTypes {
     };
 }
 
+export function previewAsset(asset: string): IAssetActionTypes {
+    return {
+        type: MANAGE_ASSETS_PREVIEW,
+        payload: asset,
+    };
+}
+
+export function onAssetPreviewPanelClosed(): IAssetActionTypes {
```

Comment:
This action name could cause confusion, as it seems this action reacts to the panel closing when in reality this action closes the panel. Have a look at the `Sets` store for example names so that we keep a common format across store branches.
https://github.com/superdesk/superdesk-client-core/tree/sams/scripts/extensions/sams/src/store/sets

Reply:
Resolved.